src/main/java/plugily/projects/thebridge/handlers/setup/components/ModeComponent.java

```diff
@@ -98,7 +99,14 @@ public void injectComponents(StaticPane pane) {
       new SetupInventory(arena, setupInventory.getPlayer()).openModes();
     }), 1, 0);
 
-    pane.addItem(new GuiItem(new ItemBuilder(Material.REDSTONE_TORCH)
+    Material torch = Material.getMaterial("AIR");
```

Comment:
Why do you call a method when later it will be overwritten anyway? Just do Material torch without any intialization.